e2e/fixtures.ts

```diff
@@ -39,10 +36,14 @@ export const test = base.extend<{ forEachTest: void }>({
 });
 
 async function initSystemWithBaseConfig(page: Page) {
```

Comment:
@geigerzaehler , could you briefly have a look at this e2e init change?

We are completely reworking the UX of the (demo) system initialization, so the user now first has to interact with a system-init-assistant view before anything else is set up and available.